tmp/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
Probably included them by accident

Reply:
also the gitkeep of tmp/

Reply:
The `/tmp` folder is for storing a temporary output from the async code generator, in order to be able to compare it with the current state, for the `check` subcommand.

I've places those files purposely there, to keep the directory, but ignore everything inside. Maybe I am missing something, please educate me what is wrong about it.

Reply:
Alternatively, the program would need to create the directory on demand, but then, when quitting early, purposely or not, leaving the created tree behind, you gain a dirty working tree, which is sad.

Reply:
I think that the best move forward is to make the script create the tmp directory via TemporaryDirectory https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory so that it is (i) cross platform and (ii) don't have to include a tmp dir on git tree or in sdist for upstream OS packager.

Reply:
It must be created inside the tree, otherwise the code formatting rules defined in `pyproject.toml` will not apply correctly.. Alternative is to use `TemporaryDirectory`, and copy over that file. Agree?

Reply:
Agreed.

Reply:
Other alternative, not considered yet: Specifying the patch to `pyproject.toml` explicitly to `black` and `isort`, when possible?

Reply:
If it's possible, then yes, would be better than cpy.

Reply:
I tried, but it did not work, so I've used the "copying" variant.